lib/handlers/api.js

```diff
@@ -88,6 +88,11 @@ export class ApiHandler {
             .post(compileHandler.handleCmake.bind(compileHandler))
             .all(methodNotAllowed);
 
+        this.handle
+            .route('/compilerLibrary/:compiler/search')
```

Comment:
+1 to @partouf 's suggestion of calling this something else...library is already used so much elsewhere. the name makes sense of course, but ... too many libraries already :D

Reply:
Changed to "compiler repository". Done